packages/esm-forms-workspace-app/src/config-schema.ts

```diff
@@ -0,0 +1,81 @@
+import { validator, Type } from '@openmrs/esm-framework';
+
+export const configSchema = {
+  formsSectionsConfig: {
+    _type: Type.Array,
+    _elements: {
+      name: {
+        _type: Type.String,
+        _description: 'Name of the section.',
+      },
+      labelCode: {
+        _type: Type.String,
+        _description: 'Label to be used to translate the section.',
+      },
+      forms: {
+        _type: Type.Array,
+        _description: 'Configuration of the forms.',
+        _elements: {
+          name: {
+            _type: Type.String,
+            _description: 'Name of form.',
+          },
+          uuid: {
+            _type: Type.String,
+            _description: 'Uuid of the form.',
```

Comment:
Do we need both of these? Isn't there a UUID type? Rather than having an element named UUID, I'd think this could just be a list of form identifiers (i.e., things we can use to uniquely identify a form in the /forms REST endpoint)...

Reply:
"forms":{
   "_type": "Type.Array",
   "_description": "Configuration of the forms, it can be used the `Form UUID` or the `Encounter Type UUID` if you have both the `Form UUID` will be used.",
   "_elements": {
      "formUuid": {
         "_type":"Type.String",
         "_description":"Uuid of the form."
      },
      "encounterTypeUuid": {
         "_type":"Type.String",
         "_description":"Uuid of the encounter type."
      }
   }
}

Reply:
I added support for the encounter type as described above.